org/kind+auditsink.org

```diff
@@ -0,0 +1,797 @@
+ 6765 1000      0:00 {simple-init.sh} /bin/bash /usr/local/bin/simple-init.sh /
+ 6766 1000      0:03 tmate -F -v -S /tmp/ii.default.target.iisocket new-session
+ 6767 1000      0:00 {simple-init.sh} /bin/bash /usr/local/bin/simple-init.sh /
+ 6768 1000      0:00 inotifywait -e create,open --format %f --quiet /tmp --moni
+ 6769 1000      0:00 tmate -S /tmp/ii.default.target.iisocket wait-for tmate-re
+ 6770 1000      0:00 emacsclient --tty /home/humacs
+ 6772 1000      0:30 emacs --daemon
+ 7008 101       0:00 nginx: master process /usr/local/nginx/sbin/nginx -c /etc/
+ 7036 101       0:14 nginx: worker process
+ 7037 101       0:10 nginx: worker process
+ 7038 101       0:10 nginx: worker process
+ 7039 101       0:10 nginx: worker process
+ 7040 101       0:10 nginx: worker process
+ 7041 101       0:10 nginx: worker process
+ 7042 101       0:10 nginx: worker process
+ 7043 101       0:10 nginx: worker process
+ 7044 101       0:00 nginx: cache manager process
+ 7564 root      3:34 local-path-provisioner --debug start --helper-image us.gcr
+369898root      0:00 /bin/sh
+372238root      0:00 /bin/sh
+374176root      0:00 /bin/sh
+383296root      0:00 /bin/sh
+405102root      0:00 /bin/sh
+406630root      0:00 /bin/sh
+409570root      0:00 /bin/sh
+417696root      0:00 /bin/sh
+417953root      0:00 /bin/sh
+447929root      0:00 [kworker/3:1-rcu]
+451487root      0:00 [kworker/1:1-cgr]
+452591root      0:00 [kworker/4:2-cgr]
+455443root      0:00 [kworker/0:2-cgr]
+457598root      0:00 [kworker/6:0-rcu]
+458314root      0:00 [kworker/5:0-rcu]
+458322root      0:00 [kworker/u16:1+e]
+458323root      0:00 [kworker/7:1-cgr]
+460736root      0:00 [kworker/2:1-rcu]
+461997root      0:00 [kworker/3:0-rcu]
+462080root      0:00 [kworker/6:1-vir]
+463108root      0:00 [kworker/5:2-vir]
+463133root      0:00 [kworker/u16:4-e]
+463192root      0:00 [kworker/4:0-cgr]
+463193root      0:00 [kworker/1:0-eve]
+463453root      0:00 [kworker/7:0-cgr]
+463909root      0:00 [kworker/2:2-cgr]
+464823root      0:00 [kworker/0:1-eve]
+467679root      0:00 [kworker/2:0-cgr]
+467680root      0:00 [kworker/4:1-eve]
+467829root      0:00 [kworker/5:1-rcu]
+468126root      0:00 [kworker/u16:0-e]
+468127root      0:00 [kworker/5:3-mm_]
+468147root      0:00 [kworker/u16:2-e]
+468209root      0:00 [kworker/1:2-rcu]
+468210root      0:00 [kworker/6:2-rcu]
+468853root      0:00 [kworker/1:3-eve]
+468954root      0:00 [kworker/6:3-rcu]
+469066root      0:00 [kworker/3:2-eve]
+469135root      0:00 [kworker/u16:3-e]
+469227root      0:00 [kworker/7:2-eve]
+469665root      0:00 containerd-shim -namespace moby -workdir /var/lib/docker/c
+469688root      0:00 /bin/sleep 9999
+469817root      0:00 /bin/sh
+470159root      0:00 /bin/sh
+470160root      0:00 bash
+470161root      0:00 ps ax
+#+end_src
+
+* Build kind image
+** go get k8s
+#+begin_src shell :async yes
+  go get -u github.com/kubernetes/kubernetes
```

Comment:
From this [documentation](https://github.com/kubernetes/kubernetes#you-have-a-working-go-environment) it looks like the following is required
```
mkdir -p $GOPATH/src/k8s.io
cd $GOPATH/src/k8s.io
```

Reply:
then switch `go get -u github.com/kubernetes/kubernetes`
for `git clone https://github.com/kubernetes/kubernetes`